### ISSUE TITLE
enable user authentication with credentials

### ISSUE THREAD

**Issue by Mogakamo** [Mogaka-tech/website-builder#24]

**Comment by precious-adeyinka** [comment]
Hey @Mogakamo , what kind of authentication are you desiring, do you mind firebase, auth0 or I think I can play around with appwrite's backend actually, although I have never used it, but would love to use this as an opportunity to try it out if possible, but nevertheless, what is your auth choice?

**Comment by Mogakamo** [comment]
@precious-adeyinka have you ever used express.js or passport.js authentication libraries?

**Comment by precious-adeyinka** [comment]
Yeah, but just once for a project really, I can look into it again, with your help, if that is your preferred auth strategy then if I become rusted with the concepts and implementation, but would be fun fiddling around again I guess.

**Comment by Mogakamo** [comment]
Ok I'll guide you through. Just waiting for my backend guy to finish setting up the project and then we can begin in a day or two

**Comment by Mogakamo** [comment]
In the mean time you can fork the repository as you wait for further guidance.

**Comment by precious-adeyinka** [comment]
Awesome, I will do that!

**Comment by Mogakamo** [comment]
Which tech stack are you proficient with for the backend side @precious-adeyinka

**Comment by precious-adeyinka** [comment]
Not so proficient in django and flask just simple projects.

So I prefer nodejs actually.

**Comment by Mogakamo** [comment]
I will switch to another issue so that we could finish with the setup and then come to this. This could result to around two pull requests that you could use for the hacktoberfest challange

**Comment by Mogakamo** [comment]
Yeah setting nodejs for the backend. 
For the frontend I am using vue js but that should not worry you much for now. What I actually need is to have a fully functional backend service then get to intergrate later with the frontend application. @precious-adeyinka

**Comment by Mogakamo** [comment]
I actually I am trying to vision the whole thing getting to work in a way that the client-side gets to consume the backend by fetching from the endpoints created by the backend side

**Comment by Mogakamo** [comment]
I havent designed the system as I am still doing a small research at the moment.
Can we switch the chat to whatsapp? 

if so this is my number +254757478812

**Comment by Mogakamo** [comment]
ok text me there and we can discuss more

**Comment by precious-adeyinka** [comment]
Already

**Comment by precious-adeyinka** [comment]
Okay switching now.
